win32ss/gdi/ntgdi/palette.c

```diff
@@ -769,13 +769,8 @@ IntGdiRealizePalette(HDC hDC)
 
     ASSERT(ppalDC->flFlags & PAL_INDEXED);
 
-    // FIXME: Should we resize ppalSurf if it's too small?
-    realize = (ppalDC->NumColors < ppalSurf->NumColors) ? ppalDC->NumColors : ppalSurf->NumColors;
-
-    for (i=0; i<realize; i++)
-    {
-        InterlockedExchange((LONG*)&ppalSurf->IndexedColors[i], *(LONG*)&ppalDC->IndexedColors[i]);
-    }
+    DPRINT1("RealizePalette unimplemented for %s\n", 
+            (pdc->dctype == DC_TYPE_MEMORY ? "memory managed DCs" : "device DCs"));
```

Comment:
Does this debug print slow down the painting? If yes, better do `DPRINT` instead.

Reply:
From my past experience, this is not called that much, so leaving this as a DPRINT1 makes sense so that it will be logged and this can help us find why something is not working correctly. If it becomes a nuisance, then it can be demoted to a DPRINT later. This was a DPRINT1 in Revision 50927 before @zefklop added the Realize Palette code, so I prefer to leave it as it is now. Thanks.